src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -127,6 +130,18 @@ public void periodic() {
         Rotation2d.fromDegrees(getHeading()),
         getLeftEncoder().getPosition(),
         getRightEncoder().getPosition());
+
+    SmartDashboard.putNumber("Left Enc", getLeftEncoder().getPosition());
+    SmartDashboard.putNumber("Right Enc", getRightEncoder().getPosition());
+
+    SmartDashboard.putNumber("Odo X", roboOdometry.getPoseMeters().getX());
+    SmartDashboard.putNumber("Odo Y", roboOdometry.getPoseMeters().getY());
+    SmartDashboard.putNumber("Odo H", roboOdometry.getPoseMeters().getRotation().getDegrees());
+
+    SmartDashboard.putNumber("Cargo D Pose X", FieldConstants.cargoD.getTranslation().getX());
+    SmartDashboard.putNumber("Cargo D Pose Y", FieldConstants.cargoD.getTranslation().getY());
+    SmartDashboard.putNumber("Cargo G Pose X", FieldConstants.cargoG.getTranslation().getX());
+    SmartDashboard.putNumber("Cargo G Pose Y", FieldConstants.cargoG.getTranslation().getY());
```

Comment:
Do we still want all these SmartDashboard outputs? Maybe remove some to decrease clutter? Though ig may still be useful while testing, in that event they could be added and removed as needed.